src/sparql/reports/gard-mondo-mapped-obsoletes.sparql

```diff
@@ -0,0 +1,17 @@
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?cls ?cls_label ?xref ?comment WHERE {
```

Comment:
New SPARQL query for this work, since it was not obvious how/if we could get the "obsoletion reason" (`rdfs:comment`) in this precise table format in OAK.